meta-resin-common/classes/image_types_resin.bbclass

```diff
@@ -78,6 +78,8 @@ python() {
         d.setVar('RESIN_RAW_IMG', '${DEPLOY_DIR_IMAGE}/${IMAGE_NAME}.rootfs.resinos-img')
         d.setVar('RESIN_DOCKER_IMG', '${DEPLOY_DIR_IMAGE}/${IMAGE_NAME}.rootfs.docker')
         d.setVar('RESIN_HOSTAPP_IMG', '${DEPLOY_DIR_IMAGE}/${IMAGE_NAME}.rootfs.hostapp-ext4')
+
+    d.setVar('RESIN_IMAGE_BOOTLOADER_DEPLOY_TASK', ' '.join(bootloader + ':do_deploy' for bootloader in d.getVar("RESIN_IMAGE_BOOTLOADER", True).split()))
```

Comment:
In what cases could we have multiple bootloaders on a board?

Reply:
See git commit log, there is a link there to such a board